src/librustdoc/config.rs

```diff
@@ -769,33 +760,38 @@ impl Options {
 
 /// Prints deprecation warnings for deprecated options
 fn check_deprecated_options(matches: &getopts::Matches, diag: &rustc_errors::Handler) {
-    let deprecated_flags = ["input-format", "no-defaults", "passes"];
+    let deprecated_flags = ["input-format"];
+
+    for &flag in deprecated_flags.iter() {
+        if matches.opt_present(flag) {
+            diag.struct_warn(&format!("the `{}` flag is deprecated", flag))
+                .note(
+                    "see issue #44136 <https://github.com/rust-lang/rust/issues/44136> \
+                    for more information",
+                )
+                .emit();
+        }
+    }
+
+    let removed_flags = ["plugins", "plugin-path", "no-defaults", "passes"];
 
-    for flag in deprecated_flags.iter() {
+    for &flag in removed_flags.iter() {
         if matches.opt_present(flag) {
-            let mut err = diag.struct_warn(&format!("the `{}` flag is deprecated", flag));
+            let mut err = diag.struct_warn(&format!("the '{}' flag no longer functions", flag));
```

Comment:
I think the backquotes were fine, no need to change them.
```suggestion
            let mut err = diag.struct_warn(&format!("the `{}` flag no longer functions", flag));
```

Reply:
I didn't change them, it matches the format used for removed flags before, but I can certainly align removed flags and deprecated flags to both use backticks.